### PR TITLE
feat: run coveralls in diff env

### PR DESCRIPTION
**Pull request by chapayevdauren**

### CODE REVIEW

File: src/scripts/upload.sh [hunk 1]
```diff
@@ -4,11 +4,20 @@ if [ "$PARALLEL_FINISHED" = true ]; then
...
 fi
+
 if [[ $EUID == 0 ]]; then export SUDO=""; else export SUDO="sudo"; fi
-$SUDO npm install -g coveralls
+
+if ! command -v npm &> /dev/null; then
+  $SUDO pip3 install coveralls
+else
+  $SUDO npm install -g coveralls
+  python -c 'print("Coveralls installed!")'
```
Comment (st4lk) [inline src/scripts/upload.sh]: Why python and not echo?

**Review by st4lk — COMMENTED** [review]
Hey Dauren, can you please follow our github PR title rules?

File: src/scripts/upload.sh (hunk 1 above)
Comment (chapayevdauren) [reply src/scripts/upload.sh]: I am testing python in node image

**Review by st4lk — APPROVED** [review]
Please follow our PR title rules (imperative mood, link to JIRA issue)

**Pull request merged**